scripts/rabbitmq-env

```diff
@@ -343,6 +343,8 @@ run_escript()
     exec "${ERL_DIR}erl" +B \
         -boot "${CLEAN_BOOT_FILE}" \
         -noinput -noshell -hidden -smp enable \
+        -kernel inet_dist_listen_min 55672 \
+        -kernel inet_dist_listen_max 55672
```

Comment:
I'll add the missing continuation character. Noticed when `gmake run-broker` started hanging without it.